lib/mongoid/shell/properties/authentication_database.rb

```diff
@@ -0,0 +1,13 @@
+module Mongoid
+  module Shell
+    module Properties
+      module AuthenticationDatabase
```

Comment:
Why isn't this called `AuthSource`?

Reply:
The naming across the `mongo` driver (`auth_source`) and command line utilities (`--authenticationDatabase`) is inconsistent. This tries to stay consistent with the command line naming, but we can update if we misinterpreted that convention.

Reply:
I see, we can leave this as is.